test/format/backup.c

```diff
@@ -50,7 +50,12 @@ check_copy(WT_SESSION *session, uint64_t id)
     g.backup_verify = true;
     wts_open(to_path, &conn, false);
     g.backup_verify = false;
-
+    /*
+     * Since RTS is not ran with precise checkpoint, we need to use prepare discover cursor to claim
+     * all pending prepared transactions.
+     */
+    if (GV(PRECISE_CHECKPOINT))
+        wts_prepare_discover(conn);
```

Comment:
I know there were talks about making this internal to WT in cases with precise ckpt? If that's still the plan should there be a fixme ticket that should be linked here?

Reply:
Afaik I don't think there's a plan for that, at least not at the moment?
@quchenhao can you confirm this please?